features/src/main/scala/com/salesforce/op/features/types/Lists.scala

```diff
@@ -78,3 +78,12 @@ object DateTimeList {
   def empty: DateTimeList = FeatureTypeDefaults.DateTimeList
 }
 
+/**
+ * A list of map values
+ */
+class MapList(val value: Seq[TextMap]) extends OPList[TextMap]
```

Comment:
Is it possible to use MultiPicklistMap instead of adding a new type?

Reply:
@feifjiang is the idea that this is a general HTML parsing format? Eg sequence of segments with a map(tag -> value)? And is this a general format? or is it specific to chats?

Reply:
@leahmcguire  This was not intended as a general HTML parsing format - it's specific to Salesforce Live Chat Transcript. The sequence only consists of the agent/customer conversation, and each textmap within that sequence is a map indicating the speaker and his/her utterance, e.g. {"speaker": "Agent Tom", "text": "Hello!"}

Reply:
@tovbinm MultiPicklistMap wouldn't be applicable here, because the values are sets while we care about the order of the utterances from live chat.

Reply:
you can always prefix values with index, e.g. `0 hello`, `1 how`, `2 are`, `3 you` ;)

I dont mind, it's just adding a new type is somewhat a hassle to maintain. wdyt? @leahmcguire @Jauntbox

Reply:
@tovbinm  I chatted with Leah this morning and got the hassle of adding a new type :) . Will try to fit my use case to an existing TMOG type.